### ISSUE TITLE
POST a list of Objects

### ISSUE THREAD

**Issue by cirocosta** [GoogleCloudPlatform/endpoints-proto-datastore#92]
**tl;dr**: _is it possible, with endpoints-proto-datastore, to receive a list with objects from a POST and insert it in the db?_

Following the samples, when building my API i didn't got how could i let the users POST a list of objects so that i could be more efficient about putting a bunch of data in the db using `ndb.put_multi`, for example.

From this comment here at [endpoints_proto_datastore.ndb.model](https://github.com/GoogleCloudPlatform/endpoints-proto-datastore/blob/master/endpoints_proto_datastore/ndb/model.py#L634) i imagine that it is not possible with how it is designed. Am i right or i am missing something?

Extending [the sample provided by endpoints](https://developers.google.com/appengine/docs/python/endpoints/getstarted/backend/write_api_post) achieved the desired with:

``` python
class Greeting(messages.Message):
    message = messages.StringField(1)

class GreetingCollection(messages.Message):
    items = messages.MessageField(Greeting, 1, repeated=True)

# then inside the endpoints.api class

    @endpoints.method(GreetingCollection, GreetingCollection,
                      path='hellogretting', http_method='POST',
                      name='greetings.postGreeting')
    def greetings_post(self, request):
        result = [item for item in request.items]
        return GreetingCollection(items=result)
```
